mono/mini/mini-windows.c

```diff
@@ -152,6 +152,52 @@ get_win32_restore_stack (void)
 
 	return start;
 }
+#elif HAVE_API_SUPPORT_WIN32_RESET_STKOFLW && TARGET_ARM64
+static gpointer
+get_win32_restore_stack(void)
+{
+	static guint8* start = NULL;
+	guint8* code;
+
+	if (start)
+		return start;
+
+	const int size = 64;
+
+	/* restore_stack (void) */
+	start = code = mono_global_codeman_reserve(size);
+
+	/* push the incoming frame pointer and return onto the stack */
+	arm_stpx_pre(code, ARMREG_FP, ARMREG_LR, ARMREG_SP, -0x10);
+
+	/* update the link pointer to the current stack */
+	arm_movx(code, ARMREG_FP, ARMREG_SP);
```

Comment:
I'm not 100% sure but I *think* you might also need to subtract 16 bytes from the stack pointer (as you stored these two registers on the stack).

Reply:
Done in 
arm_stpx_pre(code, ARMREG_FP, ARMREG_LR, ARMREG_SP, -0x10);
stp         fp,lr,[sp,#-0x10]!